test/tests_unittests/test-controller-resources.js

```diff
@@ -1,46 +1,39 @@
-/* global describe before beforeEach after it */
-/* eslint-disable */
-// Third party components
-const colors = require('colors');
+/* eslint-disable func-names, prefer-arrow-callback, no-unused-expressions */
 
+// Third party components
+require('colors');
 const chai = require('chai');
 const chaiSubset = require('chai-subset');
-chai.use(chaiSubset);
-const expect = chai.expect;
-
 const mongoose = require('mongoose');
-mongoose.Promise = require('bluebird');
-
 const Mockgoose = require('mockgoose').Mockgoose;
-const mockgoose = new Mockgoose(mongoose);
-
-require('./../../app/models/resource.js');
-
 const logger = require('winston');
-logger.level = 'error';
+const Promise = require('bluebird');
 
 // Local components
+require('./../../app/models/resource.js');
 const ResourceController = require('./../../app/controllers/resources.js');
-let controller = null;
 const MockResponse = require('./mocking/MockResponse.js');
-/* eslint-enable */
 
+// Setup
+logger.level = 'error';
+mongoose.Promise = Promise;
+chai.use(chaiSubset);
+
+// Test variables
+const mockgoose = new Mockgoose(mongoose);
+const expect = chai.expect;
 
 describe('controllers/resources.js', () => {
```

Comment:
test should use real functions declarations because otherwise bindings goes wrongly..